package.json

```diff
@@ -1,11 +1,10 @@
 {
   "private": true,
   "scripts": {
-    "install": "lerna bootstrap",
-    "postinstall": "pip install -e ."
-  },
-  "devDependencies": {
-    "lerna": "^3.13.3"
+    "install": "pip install -r requirements-development.txt",
+    "nbstripout:install": "conda install -y nbstripout; nbstripout --install",
+    "conda:install-deps": "conda install -y pymedphys --only-deps",
+    "conda:create-env": "conda create -y --name pmp python=3.7; conda activate pmp; yarn conda:installdeps; yarn nbstripout:install; yarn"
```

Comment:
Here is an example of how npm scripts are made.

I haven't tested these ones, just did them quickly, but for example running the following should achieve similar to one of the bat files:

```bash
yarn conda:create-env
```

Reply:
Sure! :) Just never done it before. I assume I can call an npm scripts from a command prompt or Cmder?

Reply:
yup :)

Just fixed the scripts above (had to get yarn installed at work and test it). Now `yarn conda:create-env` will create the conda environment, and `yarn conda:delete-env` will delete it.